plugins/i18n.js

```diff
@@ -5,7 +5,7 @@
 export default ({ app }) => {
   const locales = app.i18n.locales;
   // Set the function directly on the context.app object
-  app.i18n.isoLocale = function() {
-    return locales.find(locale => locale.code === app.i18n.locale)['iso'];
+  app.i18n.isoLocale = function(code = app.i18n.locale) {
```

Comment:
What does this change do, i.e. is it used anywhere?

Reply:
Lutz wanted me to add it, @lbiedinger can you update @rwd on this?

Reply:
this was needed for when we wanted to link to the 'default' locale. It's not used elsewhere now.

Reply:
I shall revert it back